### ISSUE TITLE
Create a Sample TestNG Framework for AOF Integrration

### ISSUE THREAD

**Issue by PathriK** [Siva-Keerthi/SampleTestNG#4]
- [x] Simple test scripts which opens browser Google searches and closes
- [x] Ability to trigger from command line
- [x] Execute the test script passed as parameter
- [ ] Browser/platform options as json string in parameters
- [x] Triggering two command prompt in parallel should run without issues in parallel

**Comment by sivakumarthiyas** [comment]
We can run the test by using below maven comment through cmd.
mvn clean test -DsuiteXmlFile=testng.xml

Need to explore on giving method/class name through command line.

**Comment by sivakumarthiyas** [comment]
Commented below sections in pom.xml



Run the test case by using below MVN Commands through CMD.

mvn surefire:test -Dtest=TestParallel -DmyBrowser=chrome    Class level

mvn surefire:test -Dtest=TestParallel#test_01 -DmyBrowser=chrome  Method level

**Comment by PathriK** [comment]
What command to use if i want to run test_01 in both TestParallel & TestParallel2 in a single command?

**Comment by sivakumarthiyas** [comment]
**mvn surefire:test -Dtest=TestParallel,TestParallel2  -DmyBrowser=chrome** - To run parallel in class level. [ Note: We can use wild card as well like TestParallel*, it will execute all the methods in the class.

If we want to run method in parallel, 
Class name # method name 1 + method name 2
**mvn test -Dtest=TestParallel#test_01+test_02  -DmyBrowser=chrome**

If we want to run specific method in the class
**mvn test -Dtest=TestParallel#test_01+test_02,TestParallel2#test_01  -DmyBrowser=chrome**

**Comment by sivakumarthiyas** [comment]
I think we can close it Keerthi.

**Comment by PathriK** [comment]
Tried to execute them in parallel and it is running fine. There is one task pending - Passing the browser and other config as JSON string to the command prompt. But i think it can be postponed and done after completing basic tasks in MEAN stack. Keeping this open for now but will move it in the ToDo section of the project
